content/community/videos.md

```diff
@@ -7,168 +7,46 @@ permalink: community/videos.html
 redirect_from: "docs/videos.html"
 ---
 
-### Introduction to React
+Videos dedicated to the discussion of React and the React ecosystem.
 
-[Tom Occhino](http://tomocchino.com/) and [Jordan Walke](https://github.com/jordwalke) introduce React at Facebook Seattle.
+A playlist of videos from React.js Conf 2017.
+<iframe title="React.js Conf 2017" width="650" height="366" src="https://www.youtube-nocookie.com/embed/playlist?list=PLb0IAmt7-GS3fZ46IGFirdqKTIxlws7e0" frameborder="0" allowfullscreen></iframe>
 
-<iframe title="Tom Occhino and Jordan Walke introduce React at Facebook Seattle" width="650" height="366" src="https://www.youtube-nocookie.com/embed/XxVg_s8xAms" frameborder="0" allowfullscreen></iframe>
-
-### Introducing React Native
-
-[Tom Occhino](https://twitter.com/tomocchino) reviews the past and present of React in 2015, and teases where it's going next.
```

Comment:
I'm a little confused by the change of the order of videos. It makes it a bit hard to tell for sure, but it looks like some videos were lost. Like this one. It's not on the page anymore.

Reply:
I tried to arrange the videos chronologically from newest to oldest. A couple of the videos from the original page are now included as part of the React.js Conf playlists. For example, Tom's video above is the first video in the React.js Conf 2015 playlist. I'll scan through the list and make sure no videos were completely removed. Does that make sense? Should I add a sentence about the ordering and playlist at the top of the videos section?

Reply:
I see that I also removed the "React and Flux: Building Applications with a Unidirectional Data Flow" video because the video it linked to was in Russian (check the embed on the current page). I found the English version of the video and will add it back in in the recommit 😁

Reply:
The ordering is probably fine. It just confused me a bit when trying to verify that no videos were dropped. I don't have a lot of time to review PRs at the moment so I just kind of want to glance through and make sure things don't regress 😄  I don't think you need to add a sentence about the wording or anything. Just make sure we don't lose a video.

Reply:
I believe the latest commit addressed your requested changes. I went back through to verify any removed videos were part of the React Conf playlists. I changed the duplicate embedded links to point to the appropriate video. I also added a title to each video to more closely match the current videos section. I'm not sure why I took them out and I believe it makes it a bit easier to read through 😄 .